src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1654,6 +1654,13 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
     return preferred_impl;
 }
 
+format layout_optimizer::get_forced_format(program_node& node) {
+    if (!_forcing_map.empty() && _forcing_map.count(node.id()) != 0) {
+        return _forcing_map.at(node.id()).first;
+    }
+    return format::any;
+}
+
```

Comment:
Please check if it is really necessary from test case.

Reply:
IMHO, It doesn't have to be a function, but using get_forced_format() function is more readable than using its implementation code directly.